poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFRun.java

```diff
@@ -911,6 +1047,29 @@ public void setFontSize(double size) {
         ctSize.setVal(bd.multiply(BigDecimal.valueOf(2)).setScale(0, RoundingMode.HALF_UP).toBigInteger());
     }
 
+
+    /**
+     * Specifies the font size which shall be applied to the currently specified complex
+     * characters in the contents of this run when displayed.
+     *
+     * <p>
+     * If this element is not present, the default value is to leave the value
+     * applied at previous level in the style hierarchy. If this element is
+     * never applied in the style hierarchy, then any appropriate font size may
+     * be used for the non-complex characters.
+     * </p>
+     *
+     * @param size The font size as number of point measurements.
+     * @see #setFontSize(int)
+     * @since POI 5.0.0
```

Comment:
Could you use `@since POI 5.2.5` on all the new public methods? Including any new public methods that have no `@since` setting.

Reply:
Done